src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -0,0 +1,112 @@
+@import "vars";
+@import "hxCheckboxFacade";
+
+//
+// ===== Mixins for <hx-checkbox-control> states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __checkboxControl($psuedo-state: null) {
+  @if $psuedo-state == focus {
+    box-shadow: $focus-glow;
+
+    &::-ms-check {
+      border-color: $cyan-700;
+    }
+
+    ~ label > hx-checkbox {
+      border-color: $cyan-700;
+      box-shadow: $focus-glow;
+    }
+  }
+  @else if $psuedo-state == hover {
+    color: $cyan-500;
+
+    &::-ms-check {
+      background-color: $cyan-50;
+      color: $cyan-500;
+    }
+  }
+}
+
+// ========== INVALID ==========
+@mixin __checkboxControl--invalid($psuedo-state: null) {
+  color: $red-900;
+
+  &::-ms-check {
+    border-width: 2px;
+    border-color: $red-900 !important;
+    color: $red-900 !important;
+  }
+
+  @if $psuedo-state == focus {
+    box-shadow: $focus-glow-invalid;
+
+    &::-ms-check {
+      border-color: $red-900;
+    }
+
+    ~ label > hx-checkbox {
+      border-color: $red-900;
+      box-shadow: $focus-glow-invalid;
+      border-width: 2px;
```

Comment:
These are facade styles. They shouldn't be here.

Reply:
Please move these styles to `hxCheckboxFacade(invalid, focus)` and apply explicitly in `src/scss/components/checkbox/_index.scss`